airflow/lineage/backend.py

```diff
@@ -25,6 +25,27 @@
 class LineageBackend:
     """Sends lineage metadata to a backend"""
 
+    def send_start(
```

Comment:
Maybe it's best to follow the existing pattern, but the `send_`  part of this name makes assumptions about what the backend is going.

```suggestion
    def on_task_isntance_start(
```

is what I would call this. WDYT?

Reply:
I'm okay with this. Also, changed `send_fail` to `on_task_instance_fail`.